### ISSUE TITLE
Add author's name in text pages near the top (for test)

### ISSUE THREAD

**Issue by berger-n** [buda-base/public-digital-library#927]
See Text page in Figma

**Comment by berger-n** [comment]
looks already done: https://library-dev.bdrc.io/show/bdr:MW00KG0545
but maybe I'm not getting your point @roopeux?

![Image](https://github.com/user-attachments/assets/a79b64ea-d3f7-49a1-a672-6e0a718e07b5)

**Comment by roopeux** [comment]
Yes I also started noticing the authors a week ago.  If nobody added them. it is weird.  Anyways, this is done.
